### DISCUSSION TITLE
Impossible to take a clear picture of a document only using the front camera

### DISCUSSION THREAD

**Discussion by ignaciostellino** [community/community#143658] [GitHub Education]
### Hi everyone,

I want to upload the pdf with the proof that I am student of my university and also as it is in spanish I will have to upload a translation near. How would all of that enters or been able to capture only using the front camera of my phone. All is on the computer as a file. Why I don't have the option to upload a pdf file?
Thanks

**Comment by chetanty** [comment]
Try converting the document to image using online tools if it doesn't let you upload a pdf. [ilovepdf](https://ilovepdf.com/pdf_to_jpg) and [adobe's convertor](https://www.adobe.com/ca/acrobat/online/pdf-to-jpg.html) should do the job.

**Reply by ignaciostellino** [reply]
@chetanty But is not letting upload an image even. Only taking a live photo. How this will be possible to be done?
Is someone from github here to give the answer?

**Comment by samuel-404** [comment]
It seems like GitHub Education only allows you to upload photos (via your phone's front camera) rather than PDF files, which can be a bit tricky if your documents are already in PDF format. Here’s how you can handle it:

Convert your PDF to images: You can use online tools like Smallpdf or ILovePDF to turn your PDF into JPEG or PNG images. Alternatively, you can take screenshots of the document on your computer and upload those.

Upload both the original and the translation: Since your document is in Spanish, make sure to also upload an image of the translation. You can take separate photos for the original document and the translated one.

Capture clear images: If you’re using your phone to upload, try to ensure good lighting and make sure the document is aligned and clear in the camera frame.

**Reply by ignaciostellino** [reply]
@samuel-404 But it seem that only you can take photos not even upload them. Am I right?

**Reply by waleeedkhalid** [reply]
try another browser

**Reply by ignaciostellino** [reply]
I am using chrome , you said that with another browser I can do upload instead of live photo?

**Reply by waleeedkhalid** [reply]
yes, you can upload an image only

**Reply by ignaciostellino** [reply]
All the community is facing the same problem if you check this thread https://github.com/orgs/community/discussions/41401 . So not sure why github don't fix it

**Comment by rutkovskii** [comment]
If you use Mac and iPhone: 
A little trick to have an easier time taking pictures when filling out the form on the Mac – you can use continuity to use iPhone's back camera to take clearer images. I had resolution issues when taking images from Mac.

How to set up continuity:
https://support.apple.com/en-us/102546
